### ISSUE TITLE
Clips missing on My Clips page for a logged-in user

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-web#1095]
This issue was reported by Brian of London. I'll need to figure out a way to reproduce the issue to be able to fix it.

**Comment by brianoflondon** [comment]
I'm getting some console errors relating to thumbnails on 3speak (which I've informed 3speak about).

This is the only one that seems to be Podverse's problem. Not sure what was playing.

```
_app-d527a4705a749dfb.js:1          GET https://api.podverse.fm/api/v1/user-now-playing-item? 404
(anonymous) @ _app-d527a4705a749dfb.js:1
e.exports @ _app-d527a4705a749dfb.js:1
e.exports @ _app-d527a4705a749dfb.js:1
u.request @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
l @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
P.forEach.e.<computed> @ main-d977f1d2acb21ba7.js:1
l @ _app-d527a4705a749dfb.js:1
a @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
l @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
P.forEach.e.<computed> @ main-d977f1d2acb21ba7.js:1
c @ _app-d527a4705a749dfb.js:1
a @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
l @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
P.forEach.e.<computed> @ main-d977f1d2acb21ba7.js:1
B @ _app-d527a4705a749dfb.js:1
a @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
(anonymous) @ _app-d527a4705a749dfb.js:1
Ui @ framework-5f4595e5518b5600.js:1
t.unstable_runWithPriority @ framework-5f4595e5518b5600.js:1
Ql @ framework-5f4595e5518b5600.js:1
Fi @ framework-5f4595e5518b5600.js:1
bi @ framework-5f4595e5518b5600.js:1
(anonymous) @ framework-5f4595e5518b5600.js:1
t.unstable_runWithPriority @ framework-5f4595e5518b5600.js:1
Ql @ framework-5f4595e5518b5600.js:1
Yl @ framework-5f4595e5518b5600.js:1
Kl @ framework-5f4595e5518b5600.js:1
ki @ framework-5f4595e5518b5600.js:1
os @ framework-5f4595e5518b5600.js:1
t.hydrate @ framework-5f4595e5518b5600.js:1
Le.h.default.createElement.callbacks @ main-d977f1d2acb21ba7.js:1
Le @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
l @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
P.forEach.e.<computed> @ main-d977f1d2acb21ba7.js:1
M @ main-d977f1d2acb21ba7.js:1
i @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
_e @ main-d977f1d2acb21ba7.js:1
Pe @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
l @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
P.forEach.e.<computed> @ main-d977f1d2acb21ba7.js:1
M @ main-d977f1d2acb21ba7.js:1
i @ main-d977f1d2acb21ba7.js:1
Promise.then (async)
M @ main-d977f1d2acb21ba7.js:1
i @ main-d977f1d2acb21ba7.js:1
Promise.then (async)
M @ main-d977f1d2acb21ba7.js:1
i @ main-d977f1d2acb21ba7.js:1
Promise.then (async)
M @ main-d977f1d2acb21ba7.js:1
i @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
we @ main-d977f1d2acb21ba7.js:1
t.initNext @ main-d977f1d2acb21ba7.js:1
2870 @ main-d977f1d2acb21ba7.js:1
r @ webpack-af0fc1fb1b57cb39.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
r.O @ webpack-af0fc1fb1b57cb39.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
n @ webpack-af0fc1fb1b57cb39.js:1
(anonymous) @ main-d977f1d2acb21ba7.js:1
_app-d527a4705a749dfb.js:1 Error in getNowPlayingItemOnServer:  Error: Request failed with status code 404
    at e.exports (_app-d527a4705a749dfb.js:1:131395)
    at e.exports (_app-d527a4705a749dfb.js:1:133837)
    at XMLHttpRequest.P (_app-d527a4705a749dfb.js:1:125956)
```

**Comment by mitchdowney** [comment]
@brianoflondon this should be fixed in v4.12.2 https://github.com/podverse/podverse-web/pull/1116
